src/commands.js

```diff
@@ -354,11 +354,83 @@ module.exports = {
   },
   githubLogin: {
     method: 'POST',
-    path: '/auth/github/login',
+    path: '{{mount_point}}{{^mount_point}}/auth/github{{/mount_point}}/login',
```

Comment:
Can you please explain the special syntax (`^mount_point` and `/mount_point`)?

Reply:
I'm not a mustache expert, so this was my stab at injecting default values in to an expression.

https://mustache.github.io/mustache.5.html#Inverted-Sections

> An inverted section begins with a caret (hat) and ends with a slash. That is, `{{^mount_point}}` begins a "mount_point" inverted section while `{{/mount_point}}` ends it.
> 
> The enclosed content will be rendered if the key doesn't exist, is false, or is an empty list.

So the approach here was to first render the value. If that value misses (i.e. renders an empty string), the inverted selection would render the enclosed content by default.

Reply:
@kr1sp1n - Was I able to answer your question (correctly)?